### ISSUE TITLE
Question & Answer for Jet Review Plugin 

### ISSUE THREAD

**Issue by Sayeed1440** [Crocoblock/suggestions#1408]
Questions & Answers are getting popular nowadays. Most of the eCommerce website is having Q&A for their products.

It would be great if we can have something similar in Jet Review.

**Comment by haobinliang** [comment]
Yes, it will be great to have it. Would be even better if the author can choose the best anser from the coments.

**Comment by shinTM** [comment]
can i get some link to example? maybe you have one

**Comment by Sayeed1440** [comment]
Great, looks like you guys are planning to add it.

Here is one website where I saw this feature:

Website with this feature: https://snowehome.com/products/linen-sheet-set?variant=12976560078947

Feature Provider: https://www.okendo.io/inspiration/

I hope this will help.
